redirector/common.py

```diff
@@ -0,0 +1,126 @@
+import os
+import json
+import logging
+
+import boto3
+from boxsdk import Client, JWTAuth
+from boxsdk.exception import BoxAPIException
+
+
+LOGGER = logging.getLogger(__name__)
+LOGGER.setLevel(logging.INFO)
+
+SECRET_ARN = os.environ["SECRET_ARN"]
+MANIFEST_TABLE_NAME = os.environ["MANIFEST_TABLE_NAME"]
+BOX_FOLDER_ID = os.environ["BOX_FOLDER_ID"]
+
+
+def get_box_client():
+    secret = _get_secret()
+
+    # TODO(eslavich): We should pick one of snake_case or camelCase
```

Comment:
I believe the pythonic way is that variables and methods are snake_case and classes and instances of classes are camelCase.